examples/websockets.html

```diff
@@ -451,10 +455,30 @@ <h4 align="center">Stream update delay</h4>
         document.getElementById(id).style.width = "0";
       }
 
+      // pause bottom for streams
+      var stream_paused = false;
+      var pause_button = document.getElementById("pause-btn");
+      function setToggleStreamPauseText() {
+        if (stream_paused) {
+          pause_button.innerHTML = "Resume";
+        } else {
```

Comment:
Can we use this to display a "loading" spinner? Some of the data files are big/interesting and starting to take a few seconds

Reply:
I like that idea, I'll improve in the target branch